packages/amplication-git-utils/src/utils/custom-error.ts

```diff
@@ -0,0 +1,13 @@
+const DEFAULT_CUSTOM_ERROR = "Custom Error";
+const NOT_IMPLEMENTED_ERROR = "Not Implemented";
+
+export class CustomError extends Error {
```

Comment:
I think function will be much more efficient here

Reply:
can you provide an example?